bin/run_rocm_test.sh

```diff
@@ -182,16 +182,34 @@ function getversion(){
     if [ "$rocmver" == "$ompextrasver" ] || [ "$rocmver" -gt "$ompextrasver" ]; then
       echo "Using ompextrasver: $ompextrasver"
       compilerver=${versions[$ompextrasver]}
+      initialver=$ompextrasver
     else
       echo "Using rocmver: $rocmver"
       compilerver=${versions[$rocmver]}
+      initialver=$rocmver
+    fi
+
+    # There may be a patch release that is not in the supported list. To prevent
+    # aggregation of all passing tests, attempt to choose the last supported
+    # version with a major minor match. For example 5.4.4 may choose a passing list
+    # for 5.4.3.
+    if [ "$compilerver" == "" ]; then
+      initialregex="([0-9][0-9])"
+      [[ "$initialver" =~ $initialregex ]]
+      majorminor=${BASH_REMATCH[1]}
+      patchreleasever=`echo $supportedvers | sed -e 's|\.||g' | grep -m 2 -o "$majorminor[0-9]" | tail -1`
+      compilerver=${versions[$patchreleasever]}
+    fi
+    if [ "$compilerver" == "" ]; then
+      echo "Error: Cannot detect compiler version or version is not supported in this script."
```

Comment:
can we make this a warning if the nPSDB is running  the script ?

Reply:
Just made it a warning for every case. Thanks.